src/awsv4.erl

```diff
@@ -42,27 +44,34 @@ headers(Credentials, #{service := Service,
     headers_(Credentials, maps:merge(Defaults, Parameters), RequestPayload).
 
 -spec headers(Credentials :: credentials(),
-                  Service :: string(), % e.g., "s3"
-                   Region :: string(), % e.g., "us-west-2"
-                     Host :: string(), % e.g., "bucketname.s3.amazonaws.com"
-                  AwsDate :: undefined | aws_datetime(), % e.g., "20170101T000000Z"
-                TargetAPI :: string() | undefined, % e.g., "DynamoDB_20120810.CreateTable"
-                   Method :: string(), % e.g., "GET", "POST"
-                     Path :: string(),
+              Service :: string(),
```

Comment:
the comments were deleted :(

Reply:
…by hand, yes… I could've just moved them but I thought they pretty much stated the obvious.